tests/dogtag/acceptance/cli-tests/pki-key-cli/pki-key-cli-template-show-kra.sh

```diff
@@ -65,22 +65,19 @@ run_pki-key-template-show-kra_tests()
 	rlPhaseStartTest "pki_tey_template_show-002: verify when invalid key-template is provided key-template-show should fail"
 	local template=InvalidTemplate
 	rlRun "pki key-template-show $template > $temp_out1 2>&1" 255,1 "Executing pki key-template-show $template"
-	rlAssertGrep "Error: /usr/share/pki/key/templates/$template.xml (No such file or directory)" "$temp_out1"
+	rlAssertGrep "Error: /usr/share/pki/key/templates/$template.json (No such file or directory)" "$temp_out1"
```

Comment:
The tests under `tests/dogtag/acceptance` are probably not used anymore so if it's too difficult to update I'd say just leave it as is.

Reply:
I have updated it anyway, but I have a PR somewhere to remove all the legacy tests which I'm tempted to just merge.